app/controllers/admissions_admin/applicants_controller.rb

```diff
@@ -1,5 +1,14 @@
 # frozen_string_literal: true
 
+class JobAndApplicants
+  attr_accessor :job, :applicants
+
+  def initialize(job, applicants)
+    @job = job
+    @applicants = applicants
+  end
+end
```

Comment:
I think you could get away with using a hash instead of create a class for this, unless you have some other specific reason for doing it like this

```
jobAndApplicants = {job: job, applicants: applicants}
jobAndApplicants[:job]
```

Reply:
I think I was slightly confused by the logic at one point and just resolved to creating a class, but would very much like to avoid that, so will try this again.